knowledge-base/common-net8-sections.md

```diff
@@ -0,0 +1,111 @@
+---
+title: Blazor Sections Don't Show in Telerik Blazor Components
+description: Troubleshoot and understand how to use Blazor SectionOutlet in Telerik Blazor components.
+type: troubleshooting
+page_title: Blazor Sections Don't Work inside Telerik Blazor Components
+slug: common-kb-net8-sections
+tags: telerik, blazor, net8, section, sectionoutlet, sectioncontent
+ticketid: 1633408
+res_type: kb
+---
+
+## Environment
+
+<table>
+    <tbody>
+        <tr>
+            <td>Product</td>
+            <td>Telerik UI for Blazor</td>
+        </tr>
+        <tr>
+            <td>Version</td>
+            <td>5.0 and above</td>
+        </tr>
+        <tr>
+            <td>.NET version</td>
+            <td>8 and above</td>
+        </tr>
+    </tbody>
+</table>
+
+
+## Description
+
+The `ToolBarButton` inside the `SectionOutlet` below doesn't render. How to use [Blazor sections](https://learn.microsoft.com/en-us/aspnet/core/blazor/components/sections) with the Telerik Blazor ToolBar and reusable `SectionOutlet` instances?
+
+>caption ToolBar with SectionOutlet
+
+<div class="skip-repl"></div>
+
+````CSHTML
+<TelerikToolBar>
+    <SectionOutlet SectionName="ToolbarOutlet"></SectionOutlet>
+</TelerikToolBar>
+````
+
+>caption SectionContent definition
+
+<div class="skip-repl"></div>
+
+````CSHTML
+<SectionContent SectionName="ToolbarOutlet">
+    <ToolBarButton>Save</ToolBarButton>
+</SectionContent>
+````
+
+
+## Cause
+
+Some of the Telerik Blazor components rely on internal [cascading values and parameters](https://learn.microsoft.com/en-us/aspnet/core/blazor/components/cascading-values-and-parameters), to send component instances and configuration settings from parent components (for example, the ToolBar) to child components (for example, the `ToolBarButton`).
+
+Matching `Section` and `Outlet` components can reside anywhere in the app, including in different files. For the [transfer of cascading values and parameters, it matters where the `SectionContent` is](https://learn.microsoft.com/en-us/aspnet/core/blazor/components/sections?view=aspnetcore-8.0#section-interaction-with-other-blazor-features).
+
+The above means that a `ToolBarButton` inside a `SectionContent` is effectively outside the ToolBar and cannot receive cascading values from it.
```

Comment:
What is "the above" referring to? If it refers to the previous sentence, place this sentence into the same paragraph. If it refers to something else, clarify it because it is too far above.